libraries/AP_BoardConfig/canbus_driver.cpp

```diff
@@ -21,18 +21,20 @@
 
   #include <AP_UAVCAN/AP_UAVCAN.h>
   #include <AP_ToshibaCAN/AP_ToshibaCAN.h>
+  #include <AP_uLandingCAN/AP_uLandingCAN.h>
 
   // To be replaced with macro saying if KDECAN library is included
   #if APM_BUILD_TYPE(APM_BUILD_ArduCopter) || APM_BUILD_TYPE(APM_BUILD_ArduPlane) || APM_BUILD_TYPE(APM_BUILD_ArduSub)
     #include <AP_KDECAN/AP_KDECAN.h>
   #endif
 
+
 // table of user settable CAN bus parameters
 const AP_Param::GroupInfo AP_BoardConfig_CAN::Driver::var_info[] = {
     // @Param: PROTOCOL
     // @DisplayName: Enable use of specific protocol over virtual driver
     // @Description: Enabling this option starts selected protocol that will use this virtual driver
-    // @Values{Copter,Plane,Sub}: 0:Disabled,1:UAVCAN,2:KDECAN,3:ToshibaCAN
+    // @Values{Copter,Plane,Sub}: 0:Disabled,1:UAVCAN,2:KDECAN,3:ToshibaCAN,4:uLandingCAN
     // @Values: 0:Disabled,1:UAVCAN,3:ToshibaCAN
```

Comment:
You should add it here too, AFAIK there is no reason that it couldn't used in Rover too.